npm-react-codemod/test/pure-render-mixin-test2.output.js

```diff
@@ -1,8 +1,9 @@
-var React = require('react/addons');
+var React = require('react');
+var shallowCompare = require('react/addons/shallowCompare');
 
 var MyComponent = React.createClass({
   shouldComponentUpdate: function(nextProps, nextState) {
-    return React.addons.shallowCompare(this, nextProps, nextState);
```

Comment:
This code is generated by https://github.com/facebook/react/blob/master/npm-react-codemod/transforms/pure-render-mixin.js. If you're going to change these, can you change that source too and make sure the tests pass?